react/src/components/AgentSelect.tsx

```diff
@@ -6,10 +6,10 @@ import Flex from './Flex';
 import ResourceNumber from './ResourceNumber';
 import { useControllableValue } from 'ahooks';
 import { Select, SelectProps } from 'antd';
-import graphql from 'babel-plugin-relay/macro';
 import _ from 'lodash';
 import React, { useDeferredValue, useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { graphql } from 'react-relay';
 import { useLazyLoadQuery } from 'react-relay';
```

Comment:
It's better to merge these into one `import`. It would be good to apply the "import/no-duplicates" rule from eslint-plugin-import and fix it with eslint's auto-fix.

```suggestion
import { graphql, useLazyLoadQuery } from 'react-relay';